src/mon/OSDMonitor.cc

```diff
@@ -15068,6 +15070,65 @@ void OSDMonitor::convert_pool_priorities(void)
   }
 }
 
+void OSDMonitor::try_disable_stretch_mode(stringstream& ss,
+     bool *okay,
```

Comment:
(I know it's a backport, so probably too late, but: why have an out 'okay' param, instead of returning a value?)

Reply:
Hi @ronen-fr 
'okay' is a pointer to a variable that is used to indicate whether the operation is successful or not, the function `try_disable_stretch_mode` is called by another Paxos service called MonmapMonitor which is in charge of processing the command `mon disable_stretch_mode`. So the `okay` is a parameter to indicate whether this command is successful or not, if not it returns an error code and also goto a part of the code where we reply with no propose to mon map changes.

Reply:
That's not what I was trying to say. I understand the role of 'okay'. What I was asking: why return 'void' and use an 'out' parameter (and there are good reasons for minimizing out params usage), instead of returning the 'is okay' value as the ret val of the function.